packages/nextjs/test/run-integration-tests.sh

```diff
@@ -61,6 +61,11 @@ for NEXTJS_VERSION in 10 11 12; do
   else
     sed -i /"next.*latest"/s/latest/"${NEXTJS_VERSION}.x"/ package.json
   fi
+
+  # Next.js v13 requires React 18.2.0
+  if [ "$NEXTJS_VERSION" -eq "13" ]; then
+    npm i --save react@18.2.0 react-dom@18.2.0
```

Comment:
Should this maybe use `yarn` instead?

Reply:
Actually, I tried running the full test suite with react 18.2.0 and it works fine, even for Next 10, so rather than doing it here, let's just change `package.json` and use that version for everything.

Reply:
I figured that you want to make sure it also still works with older versions of react, hence the package update. If you think that's not necessary I'll update the default version to 18.

Reply:
IMO let's stick with the older versions, especially to get this in so we can release. We can re-evaluate bumping the overall version later on.